sys/fs/cd9660/cd9660_mount.h

```diff
@@ -41,12 +41,12 @@
  * Arguments to mount ISO 9660 filesystems.
  */
 struct iso_args {
-	char	*fspec;			/* block special device to mount */
+	char 	* __kerncap fspec;	/* block special device to mount */
```

Comment:
I was torn on what best matched the existing style.  Having __capability after the * instead of before like other keywords is really annoying in terms of common C styles.

Reply:
I've redone most of these.  I left sys/thr.h as it was as it just seems to read better keeping
the indentation as it is, partly due to the room before the comments in the variable name columns already.